crates/tabby-common/src/index/code/mod.rs

```diff
@@ -69,21 +61,10 @@ pub fn code_search_query(
     chunk_tokens_query: Box<dyn Query>,
 ) -> BooleanQuery {
     let schema = IndexSchema::instance();
-    let corpus_query = schema.corpus_query(corpus::CODE);
-    let git_url_query = git_url_query(&query.git_url);
+    let mut subqueries = schema.source_sub_queries(corpus::CODE, &query.git_url);
```

Comment:
```suggestion
    let mut subqueries = vec![
      (Occur::Must, Box::new(ConstScoreQuery::new(schema.source_query())))
    ]
```

No need to extract `source_sub_queries`? it probably leaks the implementation details.

Reply:
Yes. But the origin `source_query` has add `corpus_query` beside it. I just keep the logic like creating `source_query`

Reply:
Although all query are composed with `Must`, I just want to keep the same logic with previous code.

Reply:
I don't get it - nested must is still must? it doesn't change the logic if with my suggested change.